src/core/Akka/Configuration/Hocon/HoconObject.cs

```diff
@@ -159,7 +159,6 @@ public string ToString(int indent)
             var sb = new StringBuilder();
             foreach (var kvp in Items)
             {
-                if (kvp.Value.AdoptedFromFallback) continue;
```

Comment:
Was this causing the print out to miss items or something?

Reply:
Yes, I have no idea why they are there, some items are marked as non printable, which doesn't make any sense, especially since codes depends on string comparison to compare 2 config instances.

Reply:
Understood, thanks